libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1035,7 +1035,10 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 						")\n";
 				}
 				else
-					indexedArgs.emplace_back(convert(arg, *paramTypes[i]));
+				{
+					solAssert(parameterTypes[i]->sizeOnStack() == 1, "");
```

Comment:
Actually, since we now ended up not using ``expressionAsType``, we don't really need this assertion anymore - so we could remove it again. I'll approve nonetheless.